### DISCUSSION TITLE
Change messagebox font family and font size

### DISCUSSION THREAD

**Discussion by lorypelli** [element-plus/element-plus#11112] [Q&A] [answered]
How can I do this?

**Comment by ryuhangyeong** [comment, marked as answer]
## Suggest

1. set global css
2. also, inline style
3. custom scss

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtYnV0dG9uIHRleHQgQGNsaWNrPVwib3BlblwiPkNsaWNrIHRvIG9wZW4gTWVzc2FnZSBCb3g8L2VsLWJ1dHRvbj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyBoIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgRWxNZXNzYWdlLCBFbE1lc3NhZ2VCb3ggfSBmcm9tICdlbGVtZW50LXBsdXMnXG5jb25zdCBvcGVuID0gKCkgPT4ge1xuICBFbE1lc3NhZ2VCb3goe1xuICAgIHRpdGxlOiAnTWVzc2FnZScsXG4gICAgbWVzc2FnZTogaCgncCcsIG51bGwsIFtcbiAgICAgIGgoJ3NwYW4nLCBudWxsLCAnTWVzc2FnZSBjYW4gYmUgJyksXG4gICAgICBoKCdpJywgeyBzdHlsZTogJ2NvbG9yOiB0ZWFsJyB9LCAnVk5vZGUnKSxcbiAgICBdKSxcbiAgICBzaG93Q2FuY2VsQnV0dG9uOiB0cnVlLFxuICAgIGNvbmZpcm1CdXR0b25UZXh0OiAnT0snLFxuICAgIGNhbmNlbEJ1dHRvblRleHQ6ICdDYW5jZWwnLFxuICAgIGJlZm9yZUNsb3NlOiAoYWN0aW9uLCBpbnN0YW5jZSwgZG9uZSkgPT4ge1xuICAgICAgaWYgKGFjdGlvbiA9PT0gJ2NvbmZpcm0nKSB7XG4gICAgICAgIGluc3RhbmNlLmNvbmZpcm1CdXR0b25Mb2FkaW5nID0gdHJ1ZVxuICAgICAgICBpbnN0YW5jZS5jb25maXJtQnV0dG9uVGV4dCA9ICdMb2FkaW5nLi4uJ1xuICAgICAgICBzZXRUaW1lb3V0KCgpID0+IHtcbiAgICAgICAgICBkb25lKClcbiAgICAgICAgICBzZXRUaW1lb3V0KCgpID0+IHtcbiAgICAgICAgICAgIGluc3RhbmNlLmNvbmZpcm1CdXR0b25Mb2FkaW5nID0gZmFsc2VcbiAgICAgICAgICB9LCAzMDApXG4gICAgICAgIH0sIDMwMDApXG4gICAgICB9IGVsc2Uge1xuICAgICAgICBkb25lKClcbiAgICAgIH1cbiAgICB9LFxuICB9KS50aGVuKChhY3Rpb24pID0+IHtcbiAgICBFbE1lc3NhZ2Uoe1xuICAgICAgdHlwZTogJ2luZm8nLFxuICAgICAgbWVzc2FnZTogYGFjdGlvbjogJHthY3Rpb259YCxcbiAgICB9KVxuICB9KVxufVxuPC9zY3JpcHQ+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)
